File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/controllers/barcharts/PanAndZoomController.java
package com.kendoui.spring.controllers.barcharts;

import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;

import com.kendoui.spring.models.ChartDataRepository;

@Controller("dataviz-bar_chart-pan_and_zoom-controller")
@RequestMapping(value="/bar-charts/")
public class PanAndZoomController {
    
    @RequestMapping(value = "/pan-and-zoom", method = RequestMethod.GET)
    public String index(Model model) {
        
        model.addAttribute("data", ChartDataRepository.PanAndZoomData());
     
        return "/bar-charts/pan-and-zoom";
    }
}

File: wrappers/mvc-6/demos/Kendo.Mvc.Examples/Controllers/Bar_Charts/Pan_And_ZoomController.cs
using Kendo.Mvc.Examples.Models;
using Microsoft.AspNetCore.Mvc;

namespace Kendo.Mvc.Examples.Controllers
{
    public partial class Bar_ChartsController : Controller
    {
        public IActionResult Pan_And_Zoom()
        {
            return View(ChartDataRepository.PanAndZoomData());
        }
    }
}
